internal/cloud/keyvault.go

```diff
@@ -42,8 +42,8 @@ const (
 	authorizerFromCLI clientAuthMethod = "AKV_AUTH_FROM_CLI"
 
 	// defaultAuthMethod is the default auth method if user doesn't provide an environment variable
-	// the default value will be authorizerFromMI
-	defaultAuthMethod = authorizerFromMI
+	// the default value will be authorizerFromCLI
+	defaultAuthMethod = authorizerFromCLI
```

Comment:
@yizha1 @FeynmanZhou Please confirm the default behavior is expected.

Reply:
Confirm the default is CLI, which is also recommended by @dtzar .